packages/ref-doc/src/experimental.ts

```diff
@@ -20,6 +20,16 @@ export async function generateLibraryDocs(
       parseOptions: { comments: true, docs: true },
     });
     const refDoc = extractRefDocs(program, namespaces);
+    if (program.diagnostics.length > 0) {
+      // FIXME: Is this the "right" way to emit these?
+      for (const diag of program.diagnostics) {
+        if (diag.severity === "error") {
+          console.error(diag.message);
+        } else {
+          console.warn(diag.message);
+        }
+      }
```

Comment:
Do we want to process.exit(1) if there is errors or warnings(fine to do that in this `experimental` helper just meant for us)

Reply:
Well, the CI picks up the emission of the warning or error as a non-zero exit code and fails, so I'm not sure it's necessary. I prefer not to exit so it would continue to actually generate the docs (with a bunch of empty strings), for example, for local testing. `process.exit(1)` would prevent that.

Reply:
Added a process.exit statement.